docs/source/how_to/make_public_view.rst

```diff
@@ -0,0 +1,53 @@
+... make a public view for an app?
+==================================
+
+To make a part of an application visible to the user without logging in omniport we can follow the following steps -
+
+First we need to add a ``public`` key in the ``config.json`` of the app with the source file for the routes of public view. Preferably, we name this file as ``public.js``.
+
+.. code-block:: json
+
+    "public": {
+        "source": "student_profile/src/public",
+        "baseUrl": "/student_profile"
+    }
+
+Then we edit the ``public.js`` file and we try to keep it like the index.js file, containing the routes of the public-views.
+
+.. code-block:: javascript
+
+    import React, { Component } from "react";
+    import { createStore, applyMiddleware, compose } from "redux";
+    import { Provider } from "react-redux";
+    import thunk from "redux-thunk";
+    import { BrowserRouter as Router, Route } from "react-router-dom";
+
+    import rootReducers from "./reducers";
+    //local imports which are meant to be open for public-view
+    import App from "./App";
+    //css files can be same as of index.js
+    import "./index.css";
+
+    export default class AppRouter extends Component {
+        constructor(props) {
```

Comment:
Not sure but I think we follow a 2-space indentation rule in JS.

Reply:
Sure i will make the suggested changes.